app/controllers/bundles_controller.rb

```diff
@@ -1,2 +1,53 @@
 class BundlesController < ApplicationController
+  def index
+    @bundles = BundlesOffer.all
+  end
+
+  def show
+    if BundlesOffer.exists?(params[:id])
+      @bundle = BundlesOffer.find(params[:id])
+    else
+      redirect_to bundles_url
+    end
+  end
+
+  def new
+    @bundle = BundlesOffer.new
+  end
+
+  def create
+    @bundle = BundlesOffer.new(bundles_offer_params)
+    if @bundle.save
+      redirect_to bundles_url
+    else
+      render :new
+    end
+  end
+
+  def edit
+    if BundlesOffer.exists?(params[:id])
+      @bundle = BundlesOffer.find(params[:id])
+    else
+      redirect_to bundles_url
+    end
+  end
+
+  def update
+    @bundle = BundlesOffer.find(params[:id])
+    @bundle.update(tasks_params)
```

Comment:
Please review  @bundle.update(tasks_params)

Reply:
Oops, I need to change that

Reply:
cool,  thanks